### ISSUE TITLE
Ident containing space results in isValid being false

### ISSUE THREAD

**Issue by WoutProvost** [nielsbasjes/dsmr-tools#4]
I've tested 2 meters, and **Iskra AM550** and an **Enexis/Sagemcom T120D**.

The code fails for the second meter, due to the identifier containing a space:
```
/Ene5\SAGEMCOM CX2000-
```

I was able to fix it by changing the regex of the **IDENT** in the **Dsmr.g4** file like so (added space in last square brackets):
```
IDENT       : '/' [a-zA-Z0-9][a-zA-Z0-9][a-zA-Z0-9] [0-9] [\\a-zA-Z0-9 _-]+ ;
```

Is it possible to publish this fix (maybe versioned 0.3.1) to maven central?

**Comment by nielsbasjes** [comment]
Can you give me a complete record I can include as a testcase?

**Comment by WoutProvost** [comment]
```
/Ene5\SAGEMCOM CX2000-

1-3:0.2.8(50)
0-0:1.0.0(181202060910W)
0-0:96.1.1(5354545454313233343530303237333137)
1-0:1.8.1(000098.508*kWh)
1-0:1.8.2(000000.000*kWh)
1-0:2.8.1(000000.917*kWh)
1-0:2.8.2(000000.000*kWh)
0-0:96.14.0(0001)
1-0:1.7.0(00.004*kW)
1-0:2.7.0(00.000*kW)
0-0:96.7.21(00933)
0-0:96.7.9(00108)
1-0:99.97.0(10)(0-0:96.7.19)(181201030814W)(0000758635*s)(180904173923S)(0000001820*s)(180707220440S)(0000024288*s)(180611204409S)(0002246666*s)(180425175248S)(0000002897*s)(180420221654S)(0000001068*s)(180226203438W)(0000952147*s)(180215192356W)(0000268881*s)(180209184337W)(0000003307*s)(180208200717W)(0000000266*s)
1-0:32.32.0(00060)
1-0:52.32.0(00044)
1-0:72.32.0(00045)
1-0:32.36.0(00000)
1-0:52.36.0(00000)
1-0:72.36.0(00000)
0-0:96.13.0()
1-0:32.7.0(227.0*V)
1-0:52.7.0(226.0*V)
1-0:72.7.0(228.0*V)
1-0:31.7.0(000*A)
1-0:51.7.0(000*A)
1-0:71.7.0(000*A)
1-0:21.7.0(00.004*kW)
1-0:41.7.0(00.000*kW)
1-0:61.7.0(00.000*kW)
1-0:22.7.0(00.000*kW)
1-0:42.7.0(00.000*kW)
1-0:62.7.0(00.000*kW)
!0B43
```

**Comment by nielsbasjes** [comment]
I published a test version ( 0.4-SNAPSHOT ) in this maven repo: https://oss.sonatype.org/content/repositories/snapshots

If you want you can also directly download it here:
https://oss.sonatype.org/content/repositories/snapshots/nl/basjes/dsmr/dsmr-parser/0.4-SNAPSHOT/

Please let me know if this work.

**Comment by WoutProvost** [comment]
Yes works, thanks!

**Comment by nielsbasjes** [comment]
I also had an ident with a '.' in it. Same problem.
Just now I released 0.4
Thanks for helping out.
